src/main/scala/com/gu/iosdeployments/Lambda.scala

```diff
@@ -107,10 +120,13 @@ object Lambda {
     logger.info("Successfully loaded configuration...")
     val result = for {
       runningDeployments <- GitHubApi.getRunningDeployments(gitHubConfig)
+      failedDeployments <- GitHubApi.getFailedDeployments(gitHubConfig)
       appStoreConnectBetaBuilds <- AppStoreConnectApi.getLatestBetaBuilds(appStoreConnectToken, appStoreConnectConfig)
       appStoreConnectProductionBuild <- AppStoreConnectApi.getLatestProductionBuilds(appStoreConnectToken, appStoreConnectConfig)
       maybeBetaDeployment = runningDeployments.find(_.environment.contains("beta"))
       maybeProductionDeployment = runningDeployments.find(_.environment == "production")
+      maybeFailedProductionDeployment = failedDeployments.find(_.environment == "production")
+      handleFailedProductionDeployment <- handlePreviouslyFailedProductionDeployment(maybeFailedProductionDeployment, appStoreConnectProductionBuild, gitHubConfig)
```

Comment:
Very minor thing: I would probably put this line last in for the `for` comprehension (and add it to the `yield` block), just to keep the ordering consistent between searching and handling.